src/scripts/main.js

```diff
@@ -1,3 +1,14 @@
 'use strict';
 
 // write code here
+const inputs = document.querySelectorAll('input');
+
+inputs.forEach(el => {
+  el.placeholder = el.name.replace(/[A-Z]/g, letter => ` ${letter}`);
+  el.placeholder = el.placeholder[0].toUpperCase() + el.placeholder.slice(1);
+
+  el.insertAdjacentHTML('beforebegin', `
+<label for="${el.id}" class="field-label">
+  ${el.name.replace(/[A-Z]/g, letter => ` ${letter}`).toUpperCase()}:
+</label>`);
```

Comment:
```suggestion
  el.insertAdjacentHTML('beforebegin', `
    <label for="${el.id}" class="field-label">
      ${el.name.replace(/[A-Z]/g, letter => ` ${letter}`).toUpperCase()}:
    </label>
  `);
```
Also. would be great to create a separate variable for formatted name